### ISSUE TITLE
Labelling/Conversion of Buttons on X14/X20

### ISSUE THREAD

**Issue by Hartinger** [FrSkyRC/ETHOS-Feedback-Community#4628]
I have a Tandem X20S and a Twin X14S, both running ETHOS 1.5.15. I found the following anomalies w.r.t. the labelling and conversion of the buttons SG/SH on the X14 and SI/SJ on the X20.

First, in the X14 ETHOS firmware the button SG is on the right side and SH on the left side. This sequence of the letters is different to the one of the X20 ETHOS firmware where the labelling of buttons SI/SJ is left to right. This should be corrected for the X14 in order to make it consistent with the X20, i.e. SG should be the left, and SH the right button. However, it needs to be acknowledged that existing setups would be impacted by such a change.

As a consequence of the above, when models are transferred between the two radios, any allocation of the left button of the source radio is converted to an allocation of the right button of the target radio, and vice versa. At least this should be corrected, if the basic labelling for the X14 needs to remain as is.

**Comment by enzo-molinari** [comment]
I don't agree with the conversion of the buttons. I have programmed many models on my X14. If the buttons are converted, I will have to reprogram all my models!

There is also the risk that after the update someone will not do the pre-flight checks and the converted buttons will cause a crash with injuries!

**Comment by Hartinger** [comment]
I agree to this point, which I have actually mentioned in my initial post.

It is perfectly ok to leave the labelling as is, and to modify only the conversation, such that left is converted to left, and right to right. How the buttons are actually labelled is not really relevant.

**Comment by Hartinger** [comment]
I think it is still worthwhile to modify the conversion of the momentary buttons, such that left is converted to left, and right to right.

**Comment by Hartinger** [comment]
I still beleive the conversion shoul be modified as suggested.
